UnitsNet.Tests/CustomCode/AreaTests.cs

```diff
@@ -49,14 +49,14 @@ public void AreaDividedByLengthEqualsLength()
         public void AreaTimesMassFluxEqualsMassFlow()
         {
             MassFlow massFlow = Area.FromSquareMeters(20) * MassFlux.FromKilogramsPerSecondPerSquareMeter(2);
-            Assert.Equal(massFlow, MassFlow.FromKilogramsPerSecond(40));
+            Assert.Equal(40, massFlow.KilogramsPerSecond);
```

Comment:
I think this would be good place to hammer down the fact the actual value might not be exactly 40. However I'm not sure as to what precision should be used: 5, 9, 15? We could hide the fact that we don't know behind the AssertEx.Equals - but we'd have to reference the other test for the tolerance.

Reply:
We have separate tests for roundtrip tolerance, this test simply verifies the arithmetic.